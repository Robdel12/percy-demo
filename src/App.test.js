import React from "react";
import ReactDOM from "react-dom";
import App from "./App";
import renderer from "react-test-renderer";
import { render } from "react-testing-library";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("matches snapshot", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("contains a paragraph", () => {
  const { getByText } = render(<App />);
  getByText("Learn React again and again.");
});
