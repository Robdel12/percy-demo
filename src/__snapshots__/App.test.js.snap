// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="App"
>
  <nav>
    <a
      href="/"
    >
      Home
    </a>
     | 
    <a
      href="/about"
    >
      About
    </a>
  </nav>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React again and again.
    </a>
  </header>
  <footer>
    Copyright 2018
  </footer>
</div>
`;
